DS/DSAssignment6/Employee.java

```diff
@@ -0,0 +1,72 @@
+import java.util.Comparator;
+
+public class Employee implements Comparable {
+
+	private Integer employeeId;
+	private String name;
+	private String address;
+
+	public Employee(Integer employeeId, String name, String address) {
+		super();
+		this.employeeId = employeeId;
+		this.name = name;
+		this.address = address;
+	}
+
+	public Integer getEmployeeId() {
+		return employeeId;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	@Override
+	public String toString() {
+		return "Employee [name=" + name + "]";
+	}
+
+	@Override
+	public int compareTo(Object o) {
+		Employee employeeNaturalOrder = (Employee) o;
+		return getEmployeeId().compareTo(employeeNaturalOrder.getEmployeeId());
+	}
+
+	@Override
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result
```

Comment:
no need of such calculation, returning id is sufficient as it will be unique (given);